samples/javascript/README.md

```diff
@@ -0,0 +1,37 @@
+# Export API Javascript Example
+
+In this JavaScript example, build durations are calculated and written out to the JavaScript console.
+
+## Setup
+
+To run this sample:
+
+1. Install [node.js].
+2. Run `npm install` within this directory.
+
+#### Create an Export API user for your Gradle Enterprise instance
+3. Create a Gradle Enterprise user with the `Export API` role as described in the [Export API Access Control] documentation.
+4. Set two environment variables locally: `EXPORT_API_USER` `EXPORT_API_PASSWORD` to match the newly created Export API user credentials.
+
+#### Running
+5. Run `npm start {server-address}` replacing `{server-address}` with your Gradle Enterprise server URL.
+
+Now start publishing builds to your Gradle Enterprise instance and watch their build durations get written to the console.
+
+#### Integrate the sample in an HTML page
+
+If using the script in an HTML page instead of a Node application, remove the first line of [index.js]:
```

Comment:
What about the credentials? I guess we should also advice to inline the credentials in the page and warn of the potential risk as a disclaimer.